src/format.js

```diff
@@ -28,7 +28,7 @@ const tokenFunctions = {
   a:    d => d.getHours() > 11 ? 'pm': 'am',
   SSS:  d => fill(3, d.getMilliseconds()),
   SS:   d => firstN(2, fill(3, d.getMilliseconds())),
-  S:    d => firstN(1, fill(3, d.getMilliseconds())),
```

Comment:
They are very nice, it means you can add new lines without worrying about add extra commas, and it means you can reorder lines with out errors